services/gateway/apis/http-version3/swagger/definitions/tokens.json

```
@@ -96,26 +96,26 @@
         "properties": {
             "data": {
                 "type": "object",
-                "description": "List of addresses ordered by their LSK balances and other known meta information, when available",
+                "description": "List of addresses ordered by their LSK balances and other known meta information, when available.",
                 "properties": {
                     "address": {
                         "type": "string",
                         "format": "address",
                         "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
-                        "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is 41 character long identifier that begins with `lsk`.\n"
+                        "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is 41 character long identifier that begins with `lsk`."
                     },
                     "balance": {
                         "type": "string",
                         "example": "10000000",
-                        "description": "Available balance"
+                        "description": "Available balance."
                     },
                     "owner": {
                         "type": "string",
-                        "example": "Top Account"
+                        "example": "Top Account."
```

This is an example. Could be left as is.
```suggestion
                        "example": "Top Account"
```

---

applied.